Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -0,0 +1,2221 @@
+category: Endpoint
+commonfields:
```

Comment:
Could you please specify here which changes you made to which commands,  and which commands did you add from scratch?